corva/network/api.py

```diff
@@ -54,20 +54,25 @@ def _init_session(api_key: str, app_name: str, max_retries: int, allowed_methods
 
         return session
 
-    def get(self, path: str, **kwargs):
-        return self._request('GET', path, **kwargs)
+    @property
```

Comment:
You should leave these as functions and call or pass parameters through directly instead of using a layer of indirection with properties and wrapped functions. See the requests library itself for an example, or just revert back to how it was.

Reply:
The properties were added with the purpose of getting proper parameter information in IDE

https://github.com/corva-ai/python-sdk/pull/16#discussion_r559421792